<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgrammerWindowClass</class>
 <widget class="QMainWindow" name="ProgrammerWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProgrammerWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="taksListWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Task :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="taskLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="addPushButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="donePushButton">
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>addPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ProgrammerWindowClass</receiver>
   <slot>addTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ProgrammerWindowClass</receiver>
   <slot>removeTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ProgrammerWindowClass</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>taksListWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>ProgrammerWindowClass</receiver>
   <slot>toggleDone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>donePushButton</sender>
   <signal>clicked()</signal>
   <receiver>ProgrammerWindowClass</receiver>
   <slot>done()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addTask()</slot>
  <slot>removeTask()</slot>
  <slot>start()</slot>
  <slot>toggleDone()</slot>
  <slot>done()</slot>
 </slots>
</ui>
